gitpod-io/dazzle
# Upgrade GeertJohan/go.rice to v1.0.3

## Description

Upgrade GeertJohan/go.rice to v1.0.3.

This upgrade resolves an panic-causing incompatibility between `go.rice` prior to v1.0.3 and `go` subsequent to 1.18.

More details on the issue and related PR to go.rice here: https://github.com/GeertJohan/go.rice/issues/177

The panic in that issue is the same panic I see when running an unpatched `dazzle`.

### More Details

The issue ultimately caused by an issue in go.zipexe, which is depended upon by go.rice. More details on the go.zipexe issue here: https://github.com/daaku/go.zipexe/issues/6

## How to test

* Expected behavior: 

1. Run `go build`
2. `./dazzle`
3. See usage.

* Unpatched behavior: 

1. Run `go build`
2. `./dazzle`
3. See panic.


## Release Notes

```release-note
* Fix panic when running `dazzle` compiled by go1.19 or later compiler.
```

## Documentation

No documentation update needed. 😄
